cmake/find/snappy.cmake

```diff
@@ -0,0 +1,13 @@
+option(USE_SNAPPY "Enable support of snappy library" ON)
```

Comment:
Please keep MISSING_INTERNAL_SNAPPY_LIBRARY check

Reply:
Let's put this and all similar discussions about auto-detection and related features into #6981 - for historical references in the future.